src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -136,6 +137,16 @@ def go_from_trigger_mozilla_central(self):
         r.raise_for_status()
         push_id = r.json()['pushid']
 
+        for zf in self.artifactsHandler.get():
+            with zipfile.ZipFile(zf, 'r') as uz:
+                for i in uz.namelist():
+                    with uz.open(i, 'r') as fl:
+                        l = fl.read().decode('utf-8').splitlines()
+                        missing_files = list(filter(lambda x: not os.path.exists(x),
```

Comment:
A list comprehension would be more Pythonic and probably more readable. Let's try.
If it is not readable, then we can go for a normal for loop.

Reply:
A list comprehension to read SF entries of the INFO file, right?

Reply:
I mean a list comprehension instead of filter.